### ISSUE TITLE
Investigate Cell Line Data Modeling

### ISSUE THREAD

**Issue by baileyckelly** [kids-first/kf-model-omop#10]
We need to evaluate what is needed to support cell line data modeling. 
Our current model is not sufficient as a cell line is derived from a sample.

**Comment by allisonheath** [comment]
Good point - I doubt these clinical data focused models will have good modeling of biospecimens (although maybe they'll surprise me). The human cell atlas is one place we could look to mimic some of their metadata information, for example here's the json schema for their cell lines: https://github.com/HumanCellAtlas/metadata-schema/blob/master/json_schema/type/biomaterial/cell_line.json and an example: https://github.com/HumanCellAtlas/metadata-schema/blob/master/examples/bundles/public-beta/HPSI_human_cerebral_organoids/cell_line_1.json
